test/Microsoft.DotNet.Cli.Utils.Tests/StreamForwarderTests.cs

```diff
@@ -39,16 +38,18 @@ public static IEnumerable<object[]> ForwardingTheoryVariations
                 return new[]
                 {
                     new object[] { "123", new string[]{"123"} },
-                    new object[] { "123\n", new string[] {"123"} },
-                    new object[] { "123\r\n", new string[] {"123"} },
+                    new object[] { "123\n", new string[] {"123", ""} },
```

Comment:
My reasoning is that if `\n` is added to the input, a line should be added to the output. `Console.WriteLine("123")` and `Console.WriteLine("123\n")` don't produce the same output, so I don't see why this should.

Another reason is that if this was the desired behavior, the fix of this issue would be more complicated.